pkg/rpc/rpc_config.go

```diff
@@ -10,9 +8,9 @@ type (
 		EnableCORSWorkaround bool   `yaml:"EnableCORSWorkaround"`
 		// MaxGasInvoke is a maximum amount of gas which
 		// can be spent during RPC call.
-		MaxGasInvoke util.Fixed8 `yaml:"MaxGasInvoke"`
-		Port         uint16      `yaml:"Port"`
-		TLSConfig    TLSConfig   `yaml:"TLSConfig"`
+		MaxGasInvoke int64     `yaml:"MaxGasInvoke"`
```

Comment:
I'm a user and I want to specify `10` GAS in my config. Not `1000000000` of something.